### PR TITLE
Native SDKs 1.1, go-textile 0.5

### PR DESCRIPTION
**Pull request by asutula**

### CODE REVIEW

File: android/src/main/java/io/textile/rnmobile/TextileBridge.java [hunk 1]
```diff
@@ -90,16 +91,31 @@ public void summary(final Promise promise) {
...
-                catch (Exception e) {
+                catch (final Exception e) {
                     promise.reject("summary", e);
                 }
             }
         });
     }
 
+    @ReactMethod
+    public void online(final Promise promise) {
```
Comment (sanderpick) [inline android/src/main/java/io/textile/rnmobile/TextileBridge.java]: new `online` method, just returns bool (I needed it for a test)

File: android/src/main/java/io/textile/rnmobile/Util.java [hunk 2]
```diff
@@ -36,6 +46,8 @@ static String feedItemDataToBase64(FeedItemData feedItemData) {
                 return encode(feedItemData.join.toByteArray());
             case LEAVE:
                 return encode(feedItemData.leave.toByteArray());
+            case ANNOUNCE:
```
Comment (sanderpick) [inline android/src/main/java/io/textile/rnmobile/Util.java]: here's an additional feed item type

**Comment by sanderpick** [comment]
No dice so far... got some quick ideas to try in the AM.

File: android/src/main/java/io/textile/rnmobile/Util.java (hunk 2 above)
Comment (asutula) [reply android/src/main/java/io/textile/rnmobile/Util.java]: Ah nice, thanks.

**Comment by sanderpick** [comment]
I updated `android-textile` to `1.1.0` and bumped this package to `2.1.0`. I think this is GTG from me @asutula.

**Comment by asutula** [comment]
Sweet. Going to merge my ios-textile branch and do a release there, pull into there and then merge and release this

**Comment by asutula** [comment]
Ok here we go

**Pull request merged**